.travis.yml

```diff
@@ -3,14 +3,14 @@ language:
     - python
     - cpp
 python:
-    - "2.7"
-    - "3.5"
-    - "3.6"
-    - "3.8"
+    - 3.5
```

Comment:
I _think_ these should be strings instead of floats.

E.g., see https://docs.travis-ci.com/user/languages/python/#specifying-python-versions